src/pages/MyPatches.tsx

```diff
@@ -1,5 +1,30 @@
 import React from "react";
+import { useLocation, useHistory } from "react-router-dom";
+import queryString from "query-string";
+import Checkbox from "@leafygreen-ui/checkbox";
 
 export const MyPatches = () => {
-  return <div>My Patches Page</div>;
+  const { search, pathname } = useLocation();
+  const history = useHistory();
+  const parsed = queryString.parse(search);
+  let showCommitQueue = parsed.commitQueue === "true";
```

Comment:
`commitQueue` should be saved inside an interface that defines query params for this page like what we did for Tasks table on the Patch page: https://github.com/evergreen-ci/spruce/blob/new-ui-feature-branch/src/types/task.ts#L10